config/schema.error.yaml

```diff
@@ -0,0 +1,567 @@
+2:
+    title: 'Cannot write to the `./app/etc/env.php` file'
+    suggestion: 'Deployment script cannot make required changes to the `/app/etc/env.php` file. Check your filesystem permissions.'
+    stage: build
+    type: critical
+3:
+    title: 'Configuration isn''t defined in the `schema.yaml` file'
+    suggestion: 'Configuration isn''t defined in the `./vendor/magento/ece-tools/config/schema.yaml` file. Check that the config variable name is correct, and that it defined.'
+    stage: build
+    type: critical
+4:
+    title: 'Failed to parse the `.magento.env.yaml` file'
+    suggestion: 'The `./.magento.env.yaml` file format is invalid. Use a YAML parser to check the syntax and fix any errors.'
+    stage: build
+    type: critical
+5:
+    title: 'Unable to read the `.magento.env.yaml` file'
+    suggestion: 'Unable to read the `./.magento.env.yaml` file. Check file permissions.'
+    stage: build
+    type: critical
+6:
+    title: 'Unable to read the `.schema.yaml` file'
+    suggestion: ''
+    stage: build
+    type: critical
+7:
+    step: refresh-modules
+    title: 'Cannot write to the `./app/etc/config.php` file'
+    suggestion: 'Deployment script cannot make required changes to the `/app/etc/config.php` file. Check your filesystem permissions.'
+    stage: build
+    type: critical
+8:
+    step: validate-config
+    title: 'Cannot read the `composer.json` file'
+    suggestion: 'Unable to read the `./composer.json` file. Check file permissions.'
+    stage: build
+    type: critical
+9:
+    step: validate-config
+    title: 'Composer.json is missing required autoload section'
+    suggestion: 'Required `autoload` section is missing from the `composer.json` file. Compare the autoload section from the Magento `composer.json` file template, and add the missing configuration.'
+    stage: build
+    type: critical
+10:
+    step: validate-config
+    title: 'The file `.magento.env.yaml` contains an option which isn''t declared in the schema, or has an invalid value or stage'
+    suggestion: 'The `./.magento.env.yaml` file contains invalid configuration. Check the error log for detailed info.'
+    stage: build
+    type: critical
+11:
+    step: refresh-modules
+    title: 'Command `/bin/magento module:enable --all` failed'
+    suggestion: 'Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
```

Comment:
this command is often failed, when composer.log is outdated. So, we may suggest to check this too.

Reply:
added